### ISSUE TITLE
Website is Down? Can't Sync Between PC and Android

### ISSUE THREAD

**Issue by febryanvaldo** [clipto-pro/Desktop#160]
Hello,

Is the website down? as i cannot sign in to [https://clipto.pro/#/__auth_e](https://clipto.pro/#/__auth_e). I dont know what application similar to this. Please help me. :(

![](https://static.kangserver.id/7koeb.png "")

**Comment by KINGLIFER** [comment]
I think  you are screwed brother. I am sure you did not do a back up because no one expected this.

**Comment by herrmann0319** [comment]
Xclipper is similar.

**Comment by febryanvaldo** [comment]
> Xclipper is similar.

Yes i know, i already a premium user long time ago, but the PC app made my PC stutters, that's why i don't use XClipper anymore.

**Comment by herrmann0319** [comment]
Interesting... I haven't tried the PC app yet. Going to give it a go this week.

**Comment by febryanvaldo** [comment]
Yeah, but it made my PC stutters. Could you confirm when you already tested it? Thanks.

**Comment by Mike15041991** [comment]
Does anyone know how to save the local clipto database and transfer it to another device?

**Comment by seanlvjie** [comment]
Really looking forward to the developers returning and fixing this issue

**Comment by herrmann0319** [comment]
This thing isn't even on the app store anymore. Idk wth is going on tbh but it certainly doesn't look like he's returning.

**Comment by GitAlexToTheGreek** [comment]
For Windows & Android/iPhone users, I have instaneous syncing back in ClipTo...

It requires minimal configuring but does necessitate installing:

Windows ("Phone Link") 
https://www.microsoft.com/en-in/windows/sync-across-your-devices

Android ("Link To Windows") 
https://play.google.com/store/apps/details?id=com.microsoft.appmanager

iPhone
Look for "For iOS users" on the same Microsoft.com link above. 

👉 In the Phone Link app on Windows, be sure to go to:

Settings/Features/Additional settings 
...and enable:

![Screenshot 2024-08-20 170648](https://github.com/user-attachments/assets/fecf4ba0-e5f9-4db4-b6f9-411d28609ec9)

**Comment by KINGLIFER** [comment]
Clipto is on your device you mean? I still have mine yes. SO you are saying it syncs what you have on clipto or it will add to clipto?

**Comment by GitAlexToTheGreek** [comment]
Edit: @KINGLIFER To clarify and answer your question more directly, _yes_, it **adds** copies as new notes. 🙂

> Clipto is on your device you mean? I still have mine yes. SO you are saying it syncs what you have on clipto or it will add to clipto?

👉 Yes, I still have the previous installations of ClipTo on my Windows 11 laptop + Android phone...

Although CT's capabilities are severely diminished since being discontinued, I was still using them as my default Universal clipboard archiver until just a few weeks ago, til we got these messages:

![Ct-pc-s](https://github.com/user-attachments/assets/481f0077-dbf4-41b2-bbb6-84b91db28899)

and

![Screenshot_20240809_210631_Clipto](https://github.com/user-attachments/assets/9e226d65-950f-410b-9a82-f54b8d4a36b1)

Those messages indicated that copies were no longer being synced via CT's own servers. 

A new solution is to install the two apps above, and ClipTo will once again save copies in real-time.

**Comment by herrmann0319** [comment]
Thank you for sharing! I actually have all of these prerequisites installed already including the copy and paste setting on. Always pretty much have.

My issue with Clipto now is even though it's battery setting is set to unrestricted I just opened it up and the last thing it copied was from July 1st plus the last thing I copied today. I was noticing it missing many days of things I copy every time I'd open it on android. That's why I have since moved to Xclipper. 

Have any idea why I'm experiencing this? ADB has been done as well. I'd much rather use Clipto.

Also, when you say it is now syncing do you mean everything from your phone to your computer? 

What about the things you copy when you are at work or not near your computer. It will sync these once reconnected to link to windows or does it only sync the things you copy while connected?

Thanks again!!

**Comment by GitAlexToTheGreek** [comment]
Hey @herrmann0319, you're welcome. 🙂

>...the last thing it copied was from July 1st plus the last thing I copied today. I was noticing it missing many days of things I copy every time I'd open it on android...

Yeah, it's still buggy on my end too but just on the mobile app side...if I make a copy somewhere on my phone, I have to often times open ClipTo or shift focus to it in order to ensure that the copy is stored.

>Also, when you say it is now syncing do you mean everything from your phone to your computer?

Yes, the windows app now notes everything, from either device, without needing to check to make sure. (That orange bug prompt still appears the first time ClipTo's accessed following a system restart. Clicking it once then removes til the next system or app restart.)

>What about the things you copy when you are at work or not near your computer. It will sync these once reconnected to link to windows or does it only sync the things you copy while connected?

Unfortunately not - since ClipTo isn't running via it's own server anymore, and the Link apps are simply enabling the universal clipboard while ClipTo passively listens & stores content, only new copies are updated in either ClipTo, only when using the same internet connection...

Example: if I edit a note on the desktop, or away at work, etc, that note won't be automatically edited on the phone (and vice versa) the way it used to work. If I were to edit a note on either side, save it and click the copy button, _then_ it would be stored anew in both devices. 

>Have any idea why I'm experiencing this? ADB has been done as well. I'd much rather use Clipto"

Not sure. Tried doing the adb process again myself the other day, but it didn't seem to change anything.

>That's why I have since moved to Xclipper."

As I write these replies, it does occur to me how compromised/limited using CT is in its current state, even with universal clipboard working again for me. 😵

I will have to check out XClipper and give it a try for sure, thank you. Other names I've seen recommended lately (but haven't tried yet) are "KDE Connect", "Clipboard Fusion", and "Join by joaapps", so there's a few alternatives out there. 

----

😕 I just wish I could learn how to save the desktop app's data and then make it compatible w/ my mobile app, or now I suppose, compatible for migrating to whichever of the new apps mentioned above I end up going with. 

This predicament is detailed here:
https://github.com/clipto-pro/Desktop/issues/162
...in case anyone's inclined to share some suggestions.
